website/src/components/MiniBlog/index.tsx

```diff
@@ -121,13 +121,20 @@ const MiniBlog: React.FC = () => {
     }
   }, [jsonMdData]);
 
-  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => 
-          <StyledTab
-            label={`${item}(${tagsDistribution[item as keyof typeof tagsDistribution]})`}
-            value={item}
-            key = {item}
-          />
-  );
+  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
```

Comment:
Where is the trimming part that we discussed. Am I missing something ?

Reply:
Trim functionality is already available for the tags

Reply:
code at getPost.js for trim tags
``` 
                if (metadata){
                    post = {
                        id: index + 1,
                        title: metadata.title || 'No title',
                        author: metadata.author || 'No author',
                        author_info: metadata.author_info || 'No author information',
                        date: metadata.date || 'No date available',
                        **tags: metadata.tags.trim().split(',') || 'No tags available',**
                        excerpt: metadata.excerpt || '',
                        content: content || 'No content available'
                    };
                }
```

Reply:
@AVRahul  -- aren't we removing the individual tags?

For eg.

```
`tag1,    tag2,    tag3.  , tag4`
```